apps/dotcms-ui-e2e/src/pages/DotSiteBrowser/DotEditPage.ts

```diff
@@ -0,0 +1,65 @@
+import Navigation from '../../support/shared/Navigation';
+import Page from '../../support/shared/Page';
+
+const URL = '/dotAdmin/#/edit-page/content';
+const IFRAME = 'iframe.dot-edit__iframe';
+const DIALOG_IFRAME = 'iframe#detailFrame';
+
+class DotEditPage {
+    static openPage(page: string) {
+        const url = `${URL}?url=${page.replace(' ', '-')}`;
+        Navigation.visit(url);
+        Navigation.assertPageUrlIs(url);
+    }
+
+    static goToMode({ type }: { type: string }) {
+        Page.click(cy.get('p-selectbutton .p-button').contains(type));
+    }
+
+    static openAddToContainerDialog({ type }: { type: string }) {
+        cy.intercept('GET', 'appconfiguration').as('appConfiguration');
+        cy.wait('@appConfiguration');
+
+        cy.get(IFRAME)
+            .iframe()
+            .then((iframe) => {
+                Page.click(cy.get(iframe).find('#section-1 .dotedit-menu__button'));
+                Page.click(cy.get(iframe).find('#section-1 .dotedit-menu__item').contains(type));
+            });
+    }
+
+    static addElementToContainer({ elementIndex }: { elementIndex: number }) {
+        cy.intercept('POST', 'ContentletAjax.searchContentlets.dwr').as('searchContentlets');
+        cy.wait('@searchContentlets');
+        cy.get(DIALOG_IFRAME)
+            .iframe()
+            .then((iframe) => {
+                Page.click(cy.get(iframe).find(`#rowId${elementIndex} #${elementIndex}`));
+            });
+    }
+
+    static checkToastMessage({ message }: { message: string }) {
+        Page.assertElementSize('p-toastitem', 1);
+        Page.assertElementContains('p-toastitem dot-icon + span', message);
+    }
+
+    static checkContainerMustHaveElements({
```

Comment:
This method works to make sure you have something in the container, but we need a method to validate the content that was added is the one that you selected

Reply:
agree, this and more validation/verification can def be defined later on, when the FE in collaboration with QA define the use case scenarios that QA think more valuable to test.

Right now it's a first approach of what Cypress is capable of doing